docs/product-migration-guide/src/main/asciidoc/migration-verify-java-proc.adoc

```diff
@@ -0,0 +1,19 @@
+[id='migration-verify-java-proc']
+= Verify the Java project migration
+
+After you have migrated all Java project data and have upgraded the relevant database, rebuild the Java project to verify successful migration.
+
+.Prerequisites
+* Java project dependencies have been updated.
+* The {PRODUCT_OLD} 6.4 database to be used with {PRODUCT} {PRODUCT_VERSION} has been upgraded.
+
+.Procedure
+. Rebuild each Java project using a regular Maven build.
++
+[source]
+----
+$ ~/PROJECT_DIR/mvn clean install
```

Comment:
I'm confused about this example. I don't think most people have maven installed in their project. Usually it's somewhere on their system (for example, on my Fedora workstation it's `/usr/bin/mvn`).

Reply:
Fixed, per our convo.